src/foundation/src/PDFsharp/src/PdfSharp/Pdf.Signatures/DefaultSigner.cs

```diff
@@ -18,6 +19,31 @@ public DefaultSigner(X509Certificate2 Certificate)
             this.Certificate = Certificate;
         }
 
+        public byte[] GetSignedCms(Stream stream, int pdfVersion, byte[] timestampToken)
```

Comment:
This method is pretty much identical to the 2-parameters signature. You could make the 2-params method calls the 3-params one with a null timestampToken, it would make less code to maintain.

Only difference I see is the 'silent' parameter of signedCms.ComputeSignature(signer, false). Do you need this param equal to 'false'?